deploy/addons/storage-provisioner/storage-provisioner.template

```diff
@@ -51,7 +51,7 @@ spec:
   hostNetwork: true
   containers:
   - name: storage-provisioner
-    image: {{default "gcr.io/k8s-minikube" .ImageRepository}}/storage-provisioner:v1.8.1
+    image: {{default "gcr.io/k8s-minikube" .ImageRepository}}/storage-provisioner-IMAGETAG_NONE:v1.8.1
```

Comment:
Why is it necessary to add -IMAGETAG_NONE here? Can't this image name just be left alone?

Reply:
Some images used in templates does not have a amd64 archTag
Example: `gcr.io/k8s-minikube/storage-provisioner-IMAGETAG_NONE:v1.8.1` will become `gcr.io/k8s-minikube/storage-provisioner:v1.8.1`